lib/td/logger/version.rb

```diff
@@ -1,7 +1,7 @@
 module TreasureData
 module Logger
 
-VERSION = '0.3.28'
+VERSION = '0.3.29'
```

Comment:
[suggestion] If we are going to drop old Ruby versions, it seems better for the next version to be a minor update instead of a patch one, e.g.

```suggestion
VERSION = '0.4.0'
```